articles/migrations/guides/liveid-api-deprecation.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Migration to LinkedIn API V2
+description: This article covers the LinkedIn API deprecation and how to update your Auth0 LinkedIn Connection.
+toc: true
+contentType:
+  - how-to
+useCase:
+  - add-login
+  - migrate
+---
+
+# Microsoft Account Migration to Azure AD (personal accounts) + Microsoft Graph
+
+In October 2017, Microsoft announced the [deprecation of the Live Connect API and Live SDK](https://developer.microsoft.com/en-us/office/blogs/outlook-rest-api-v1-0-office-365-discovery-and-live-connect-api-deprecation). This is a Microsoft deprecation that will affect Auth0 users using the Microsoft social connection, and requires changes in the Auth0 configuration and potentially application code. 
+
+The change implies switching:
+
+- from the Live Connect API to the Azure Active Directory API to authenticate
+- from the Live SDK to Microsoft Graph to be able to get other resources including user profiles, contacts, files, etc
+
+You can decide if Auth0 uses Live Connect + Live SDK or Azure AD + Microsoft Graph using the 'Strategy Version' field in the Microsoft Account connection settings page. 
```

Comment:
I think we are already committed with the linkedin one, but after reviewing some historical talks, I believe the the `strategy_version` thing was never intended to be visible for customers. It's a bit odd to call it 'Strategy Version' since it doesn't have much meaning for a customer IMO.